operator-pipeline-images/operatorcert/static_tests/community/bundle.py

```diff
@@ -53,6 +55,18 @@ def check_osdk_bundle_validate(bundle: Bundle) -> Iterator[CheckResult]:
             yield Warn(output_message)
 
 
+def check_osdk_bundle_validate_operatorhub(bundle: Bundle) -> Iterator[CheckResult]:
+    """Run `operator-sdk bundle validate` using operatorhub settings"""
+    yield from run_operator_sdk_bundle_validate(bundle, "name=operatorhub")
```

Comment:
This is the test suite that @mporrato already added a couple of weeks ago. I removed the community test suite because as I mentioned in the comment the community test suite is deprecated and will be removed from operator-sdk soon.